applications/GeoMechanicsApplication/tests/cpp_tests/test_three_dimensional_stress_state.cpp

```diff
@@ -0,0 +1,115 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Marjan Fathian
+//
+
+#include "containers/model.h"
+#include "custom_elements/three_dimensional_stress_state.h"
+#include "includes/checks.h"
+#include "testing/testing.h"
+#include "tests/cpp_tests/test_utilities/model_setup_utilities.h"
+#include <boost/numeric/ublas/assignment.hpp>
+
+using namespace Kratos;
+
+namespace Kratos::Testing
+{
+
+KRATOS_TEST_CASE_IN_SUITE(ThreeDimensionalStressState_CalculateBMatrixReturnsCorrectResults, KratosGeoMechanicsFastSuite)
+{
+    auto stress_state = std::make_unique<ThreeDimensionalStressState>();
```

Comment:
Similar to @avdg81's comment for the plane strain test, this one can also be p_stress_state

Reply:
Has become p_stress_state_policy.